docs/book/src/developer/architecture/controllers/support-multiple-instances.md

```diff
@@ -0,0 +1,33 @@
+# Support running multiple instances of the same provider
+
+In releases <= v1alpha3 the need of supporting multiple credentials was addressed by running multiple
+instances of the same provider, each one with its own credentials.
+
+However, running multiple instances of the same provider proved to be complicated for several reason like e.g.
+
+- Complexity in packaging providers, due to the requirement to split CRDs and the web-hooks from the controller given that
+  only the controller can actually be installed many times, while CRDs and web-hooks can exist only in one instance.
+- Complexity in deploying providers, due to the requirement to ensure consistency of the management cluster e.g. avoid
+  controllers watching for the same namespaces, etc. 
+- The introduction of the concept of management groups in clusterctl, with impacts on the user experience/documentation.
+- Complexity in managing co-existence of different versions of the same provider while there could be only
+  one version of CRDs and web-hooks. 
+- A residual risk while running different version of the same provider, because some version of the provider
+  de-facto were forced to run with CRDs and web-hooks deployed from a different version.
```

Comment:
```suggestion
```

Either remove or include in the first point, seems like a duplicate?

Reply:
re- arranged in one sentence